packages/core/src/controllers/subscriber.ts

```diff
@@ -399,6 +425,17 @@ export class Subscriber extends ISubscriber {
     this.onBatchSubscribe(result.map((id, i) => ({ ...subscriptions[i], id })));
   }
 
+  private async batchFetchMessages(subscriptions: SubscriberTypes.Params[]) {
+    if (!subscriptions.length) return;
+    this.logger.trace(`Fetching batch messages for ${subscriptions.length} subscriptions`);
+    const response = (await this.rpcBatchFetchMessages(subscriptions)) as {
```

Comment:
Why are we casting? Can we attach the expected return type to `rpcBatchFetchMessages` directly?